llvm/test/tools/llvm-objcopy/ELF/cross-arch-headers.test

```diff
@@ -129,6 +134,7 @@ FileHeader:
   # Arbitrary values that do not match any value we convert to via --output-format.
   Machine:         EM_NONE
   OSABI:           ELFOSABI_STANDALONE
+  Flags:           [[FLAGS=<none>]]
```

Comment:
`YAML:137:20: error: expected sequence of bit values
  Flags:           <none>
                   ^~~~~~
yaml2obj: error: failed to parse YAML input: Invalid argument

--
`

Reply:
@Pierre-vh tried out 0 instead of /<none/> for the flag but saw the same error

Reply:
Flags is an array. It should be `[]` for the default value. It should also be fairly straightforward to implement `<none>` support for the Flags field in yaml2obj, if desired, but I don't think it's strictly needed.

Reply:
@aakanksha555 I think I understand the issue a bit better. Try this instead:
```suggestion
  Flags:           [ [[FLAGS]] ]
```

Then in the CL opt, remove the `[]`. That should expand to a `[]` when the flag isn't defined I believe.

Reply:
Please take a look at "llvm/test/tools/obj2yaml/ELF/DWARF/debug-str.yaml" for an example of how to set the Flags field in an ELF YAML input based on a variable input string. For example `-DFLAGS="[ SHF_MERGE, SHF_STRINGS ]"` in the yaml2obj command-line sets those two flags.